src/rpc/handlers/Ledger.cpp

```diff
@@ -45,6 +45,13 @@ doLedger(Context const& context)
         diff = params.at("diff").as_bool();
     }
 
+    if (params.contains(JS(full)) && params.contains(JS(accounts)))
+    {
+        if ((params.at(JS(full)).as_bool() &&
```

Comment:
I wonder if this will crash if i send a `full` or `accounts` that is not a `bool`?